content/docs/configuration.md

```diff
@@ -124,6 +124,15 @@ doing the `propose-update` command) - if you are using a different tagging
 scheme, let's say `v1.2.3` you can then set this parameter to `v{version}` and
 packit will fill in the version argument.
 
+#### archive_root_dir_template
+
+(string) If packit is not able to extract the archive root directory name directly
+from the archive, it is possible to specify it explicitly with this option. 
+
+You can use following tags in string:
+* `{upstream_pkg_name}` - upstream package name
+* `{version}` - package version
+
```

Comment:
nice write-up, I'd also mention the default, add an example and say this being utilized in `%autosetup -n ...`, so that it's perfectly clear what this is about

Reply:
thanks for the feedback, is ok now?